gradle/libs.versions.toml

```diff
@@ -3,7 +3,7 @@
 
 [versions]
 ably-chat = "0.0.1"
-ably = "1.2.45"
+ably = "1.2.46"
```

Comment:
Did we publish `1.2.46` without official release?

Reply:
Maybe, we should release snapshot instead https://stackoverflow.com/questions/23748354/gradle-add-dependency-to-github-project-to-dev-branch-instead-of-master.
This can cause issues for existing users using `1.2.*` in their project.

Reply:
No, I published to local maven :)